pom.xml

```diff
@@ -77,7 +77,7 @@
     -->
     <javadoc.exec.goal>javadoc</javadoc.exec.goal>
 
-    <slf4jVersion>1.7.7</slf4jVersion>
+    <slf4jVersion>1.7.25</slf4jVersion>
```

Comment:
> If I use slf4j apis I can pick up newer than the core uses.

Again this library is not packaged or exported to plugins AFAICT, it is just used internally.